.github/workflows/test-ui.yml

```diff
@@ -42,6 +42,9 @@ jobs:
           path: "~/Library/Caches/ms-playwright/"
```

Comment:
Is this path correct? https://github.com/defenseunicorns/zarf/actions/runs/4040760238/jobs/6946708240#step:8:19

Reply:
updated